File: csharp/src/PosIntegration.Service/Adyen/Model/AdyenRefundResponse.cs
namespace PosIntegration.Service.Adyen.Model
{
    public class AdyenRefundResponse
    {
        public string PspReference { get; set; }
        public string Response { get; set; }
        public string ErrorCode { get; set; }
        public string Message { get; set; }
        public string RawBody { get; set; }

        public bool IsSuccess()
        {
            return "[refund-received]".Equals(Response);
        }
    }
}

File: java/src/main/java/com/pos/integration/service/adyen/model/AdyenRefundResponse.java
package com.pos.integration.service.adyen.model;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class AdyenRefundResponse {

    private final String pspReference;
    private final String response;
    private final String errorCode;
    private final String message;
    private final String rawBody;

    public boolean isSuccess() {
        return "[refund-received]".equals(response);
    }
}
